### DISCUSSION TITLE
video thumbnail next-gen format

### DISCUSSION THREAD

**Discussion by paulhaem** [pimcore/pimcore#15481] [Ideas]
As of now the ffmpeg adapter generates the thumbnail as `png`: https://github.com/pimcore/pimcore/blob/11.x/lib/Video/Adapter/Ffmpeg.php#L215

Google Lighthouse recommends to use next-gen formats like `webp`. Are there any disadvantages in using `webp` instead of `png` for video thumbnails? This would also unlock the possibility to use animated video thumbnails for future features.

**Comment by dvesh3** [comment]
@paulhaem I believe this is just due to legacy reasons. It would be nice to receive a PR for this upgrade 😊

**Reply by paulhaem** [reply]
Great, I'll take a look as soon as I have some time.
